l10n_ro_stock_report/report/stock_report.py

```diff
@@ -0,0 +1,321 @@
+# Copyright (C) 2020 NextERP Romania
+# Copyright (C) 2020 Terrabit
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+import logging
+
+from dateutil.relativedelta import relativedelta
+
+from odoo import api, fields, models
+
+_logger = logging.getLogger(__name__)
+
+
+class DailyStockReport(models.TransientModel):
+    _name = "stock.daily.stock.report"
+    _description = "DailyStockReport"
+
+    # Filters fields, used for data computation
+
+    location_id = fields.Many2one(
+        "stock.location",
+        domain="[('usage','=','internal'),('company_id','=',company_id)]",
+        required=True,
+    )
+
+    found_product_ids = fields.Many2many(
```

Comment:
ar trebui 2 campuri, cum era inainte, 1 pentru ce poate user-ul sa faca input, altul daca nu a selectat nici un produs, dupa ce ii dai show card, wizard-ul ramane vizibil, cu man2many tags afiseaza toate produsele din found, daca sunt 200 de produse o sa strice layout-ul si nu mai vezi butoanele
deci product_ids = ce poate user-ul sa faca input; found_product_ids = sa le ia pe toate ce au miscari/cantitati daca user-ul nu selecteaza, vizibil in wizard doar product_ids

Reply:
ar trebui sa avem cele doua campuri Many2many pe product.product ce am vazut ca nu merge ( imi da eroare) dar pe varianta  acesta functioneza , found_product_ids se populeaza cu ce alege userul si ramane cu produsle selectate sau se pouleaza cu produsele care sunt in  locatie si au miscari( populara este dupa ce se inchide wizardul)

Reply:
Merge dacă definești tu tabelul de many2many și coloanele pentru fiecare camp